MakariiHlyva/HW3/Practical Task1/main.py

```diff
@@ -0,0 +1,22 @@
+#---------------Work with string of Zen of Python file----------------------------
+
+file = open("python.txt", "r")
+data_from_file = file.read()
+
+# Find quantity of different word is file
+quantity_of_better = data_from_file.find('better')
```

Comment:
method find() return index of element
you need to use method count()

Reply:
ok, thanks))